core-primitives/test/Cargo.toml

```diff
@@ -28,10 +28,12 @@ itp-sgx-crypto = { path = "../sgx/crypto", default-features = false }
 itp-stf-state-handler = { path = "../stf-state-handler", default-features = false }
 itp-teerex-storage = { path = "../teerex-storage", default-features = false }
 itp-time-utils = { path = "../time-utils", default-features = false }
-itp-types = { path = "../types", default-features = false }
 itp-ocall-api = { path = "../ocall-api", default-features = false }
 itp-storage = { path = "../storage", default-features = false, features = ["test"] }
 
+# integritee
+itp-types = { git = "https://github.com/integritee-network/pallets.git", branch = "gk/types_and_utils", default-features = false, features = ["test"] }
```

Comment:
Why do we need the `test` feature in a non-test environment here?

Reply:
For the same reason why itp-storage has the feature test. It activates the definition of the mocks.